README.md

```diff
@@ -1,70 +1,176 @@
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size](https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size)
+NO installation required
 
-### Making a Progressive Web App
+### Usage
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app](https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app)
+You can use it as a book organizer for your own book shelves
+or for small book store
 
-### Advanced Configuration
+### Run tests
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
+To run tests:
+Just check for linter, GH-workflows
 
 ### Deployment
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
+No deployment required
 
-### `npm run build` fails to minify
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+# 👥 Author <a name="author"></a>
+
+> Collaborators of this project.
+
+👤 Mark Yegon
+- GitHub: [MarkYegon7](https://github.com/MarkYegon7)
+- Email: [markyegonk@gmail.com]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Some future features gonna be added soon
+
+- [ ] **Sign up online**
+- [ ] **Newsletter Email**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+👤 **Contributors**
+
+> thanks for your effort, time and valuable contributions
+- 
+> Contributions, issues, and feature requests are welcome!
+
+
+🔭 **Issues**
+
+> Feel free to check the 
+[issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+> Write a message to encourage the Author to continue development
+
+If you like this project your comments, messeges, and support are welcome.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> special thanks.
+
+Thank you,
+> to all code reviewers who guid us through this project.
+
+> to all cool people who master techs and still answeres tech questions.
+
+> to everyone who support us, inspired us, and teach us.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+> questions that new developers may ask.
+
+- **Can I use it as a template for my Project?**
+
+  - yes you can use it, edit it and distribute it.
+
+- **how to edite it ?**
+
+  - if you know html, css, and Javascript you can easily edit it with any code editor
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 📝 License <a name="license"></a>
+
+This project is under [MIT](./LICENSE.md) licensed.
+
```

Comment:
- [ ] The License link is broken, Please make sure to fix this. It will make the README file more professional.